src/lambda/commands/importLambda.ts

```diff
@@ -155,7 +155,8 @@ async function downloadAndUnzipLambda(
                     resolve()
                 })
                 .on('error', err => {
-                    throw err
+                    // Throw sanitized error so we don't reveal a potentially active URL to private resources.
+                    throw new Error('Error downloading Lambda code from AWS.')
```

Comment:
mentioning the name or arn should be ok though.
```suggestion
                    throw new Error(`Error downloading Lambda code from AWS: ${functionArn}`)
```

Reply:
Irrelevant; handled in the catch statement further down in the file.